dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -260,6 +260,14 @@ export const Picture = ({
 			};
 		});
 
+	const ratio = parseInt(height, 10) / parseInt(width, 10);
+	/**
+	 * The assumption here is readers on devices that do not support srcset
+	 * are likely to be on poor network connections so we're going
+	 * to fallback to the smallest image.
+	 */
+	const [fallbackSource] = sources.slice(-1);
```

Comment:
Could we mention the order of the sources in the comment?

Reply:
Completely sideline - [but could we use](https://caniuse.com/mdn-javascript_builtins_array_at) [Array.prototype.at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at)? It is available at [polyfill.io](https://polyfill.io/) - but not sure what our rules are to adding to that is.

Reply:
Yes, this could be a valuable refactor.

It’s only used in three instances in the codebase, so I don’t think it’s major. One blocker is that I could not get ESLint to understand that the result was indeed a `string | undefined` and not just an `any`…

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/76776/192483056-cda43536-24d3-4f0e-a3fe-69b151117bcc.png">